frontend/lib/norent/about.tsx

```diff
@@ -68,34 +70,42 @@ export const PartnerLogos = () => (
 );
 
 export const NorentAboutPage: React.FC<{}> = () => (
-  <Page title="About" className="content">
+  <Page title={li18n._(t`About`)} className="content">
     <section className="hero is-medium">
       <div className="hero-body">
         <div className="container jf-has-text-centered-tablet">
-          <h2 className="title is-spaced has-text-info">About</h2>
+          <h2 className="title is-spaced has-text-info">
+            <Trans>About</Trans>
+          </h2>
           <br />
           <p className="subtitle">
-            Learn about why we made this tool, who we are, and who our partners
-            are.
+            <Trans>
+              Learn about why we made this tool, who we are, and who our
+              partners are.
+            </Trans>
           </p>
         </div>
       </div>
       <br />
       <div className="container jf-has-centered-images jf-space-below-2rem">
-        <JumpArrow to="#more-info" altText="Learn more" />
+        <JumpArrow to="#more-info" altText={li18n._(t`Learn more`)} />
         <br />
       </div>
     </section>
 
     <section className="hero" id="more-info">
       <div className="hero-body">
         <div className="container jf-has-text-centered-tablet">
-          <h2 className="title is-spaced">Why We Made This</h2>
+          <h2 className="title is-spaced">
+            <Trans>Why We Made This</Trans>
```

Comment:
Yeah good point! For some reason, these were in title case in the designs but I'm sure that was just an oversight as we decided on sentence case

Reply:
Actually, on the second thought, let's check in with Tahnee about this because the designs use a mix of title case and sentence case and there may be intention behind it.